### DISCUSSION TITLE
Can't tell if I'm making a mistake or if there's a bug

### DISCUSSION THREAD

**Discussion by omnosti** [st3v3nmw/obsidian-spaced-repetition#369] [Q&A]
I have a markdown file with the #flashcards tag and a few multiline flashcards. The cards themselves work fine when I choose "cram flashcards in this note" from the command palette, but I can't get anything useful to happen with the Review button in the status bar nor with the "Review flashcards" ribbon button. The status bar button pops up a input field that returns zero results no matter what I enter (title, content, tags, etc), and the pop-up from the ribbon button only lets me expand and collapse the #flashcards tag. If I click a nested tag under #flashcards, the list collapses and nothing else happens. This behavior is consistent regardless of whether or not I have cards due (opening the note that contains the due cards makes them *not* due, I'd guess this is intended).

Here is the note I'm testing the plugin with:
![image](https://user-images.githubusercontent.com/96652976/148604445-1fe26d9f-f3b5-47cf-9892-6fbe28b9ea7f.png)

I have a feeling I'm making a mistake somewhere, but I read the documentation and I still can't figure out what's wrong. Thanks for reading!

**Comment by st3v3nmw** [comment]
Likely a bug, I'll look into this.
